web/static/scss/_components.scss

```diff
@@ -115,6 +115,10 @@
   a.btn {
     margin: 0.7em 0 3em 0;
   }
+
+  form {
```

Comment:
Maybe it should be moved to the resets, then. I honestly can't remember now why I added this. Something wasn't displaying correctly :wink: